users/nazu/oled/oled_luna.c

```diff
@@ -0,0 +1,534 @@
+            0x00, 0x80, 0x80, 0x40,
+            0x40, 0x2c, 0x14, 0x04,
+            0x08, 0x90, 0x18, 0x04,
+            0x08, 0xb0, 0x40, 0x80,
+            0x00, 0x00, 0x00, 0x00,
+            0x00, 0x03, 0x04, 0x08,
+            0x10, 0x11, 0xf9, 0x01,
+            0x01, 0x01, 0x01, 0x01,
+            0x01, 0x00, 0x00, 0x00,
+            0x00, 0x00, 0x00, 0x00,
+            0x00, 0x80, 0xc0, 0x48,
+            0x28, 0x2a, 0x10, 0x0f,
+            0x20, 0x4a, 0x09, 0x10,
+            0x00, 0x00, 0x00, 0x00,
+            0x00, 0x00, 0x00, 0x01,
+            0x02, 0x0c, 0x10, 0x20,
+            0x28, 0x37, 0x02, 0x02,
+            0x04, 0x08, 0x10, 0x26,
+            0x2b, 0x32, 0x04, 0x05,
+            0x06, 0x00, 0x00, 0x00,
+            0x00, 0x00, 0x00, 0x00
+        }};
+
+    /* Sneak */
+    static const char PROGMEM sneak[2][ANIM_SIZE] =
+    {/* 'sneak1', 32x22px */
+        {
+            0x00, 0x00, 0x00, 0x00,
+            0x00, 0x00, 0x80, 0x40,
+            0x40, 0x40, 0x40, 0x80,
+            0x00, 0x00, 0x00, 0x00,
+            0x00, 0x00, 0xc0, 0x40,
+            0x40, 0x80, 0x00, 0x80,
+            0x40, 0x80, 0x00, 0x00,
+            0x00, 0x00, 0x00, 0x00,
+            0x00, 0x00, 0x00, 0x00,
+            0x1e, 0x21, 0xf0, 0x04,
+            0x02, 0x02, 0x02, 0x02,
+            0x03, 0x02, 0x02, 0x04,
+            0x04, 0x04, 0x03, 0x01,
+            0x00, 0x00, 0x09, 0x01,
+            0x80, 0x80, 0xab, 0x04,
+            0xf8, 0x00, 0x00, 0x00,
+            0x00, 0x00, 0x00, 0x00,
+            0x00, 0x00, 0x03, 0x1c,
+            0x20, 0x20, 0x3c, 0x0f,
+            0x11, 0x1f, 0x02, 0x06,
+            0x18, 0x20, 0x20, 0x38,
+            0x08, 0x10, 0x18, 0x04,
+            0x04, 0x02, 0x02, 0x01,
+            0x00, 0x00, 0x00, 0x00
+        },
+
+        /* 'sneak2', 32x22px */
+        {
+            0x00, 0x00, 0x00, 0x00,
+            0x00, 0x00, 0x80, 0x40,
+            0x40, 0x40, 0x80, 0x00,
+            0x00, 0x00, 0x00, 0x00,
+            0x00, 0x00, 0xe0, 0xa0,
+            0x20, 0x40, 0x80, 0xc0,
+            0x20, 0x40, 0x80, 0x00,
+            0x00, 0x00, 0x00, 0x00,
+            0x00, 0x00, 0x00, 0x00,
+            0x3e, 0x41, 0xf0, 0x04,
+            0x02, 0x02, 0x02, 0x03,
+            0x02, 0x02, 0x02, 0x04,
+            0x04, 0x02, 0x01, 0x00,
+            0x00, 0x00, 0x04, 0x00,
+            0x40, 0x40, 0x55, 0x82,
+            0x7c, 0x00, 0x00, 0x00,
+            0x00, 0x00, 0x00, 0x00,
+            0x00, 0x00, 0x3f, 0x20,
+            0x30, 0x0c, 0x02, 0x05,
+            0x09, 0x12, 0x1e, 0x04,
+            0x18, 0x10, 0x08, 0x10,
+            0x20, 0x28, 0x34, 0x06,
+            0x02, 0x01, 0x01, 0x00,
+            0x00, 0x00, 0x00, 0x00
+        }};
+
+    /* animation */
+    void animate_luna(void) {
+        /* jump */
+        if (isJumping || !showedJump) {
+            /* clear */
+            oled_set_cursor(LUNA_X, LUNA_Y + 2);
+            oled_write("     ", false);
```

Comment:
Just a heads up, you should use `oled_write_P` whenever possible, on AVR.  The difference is that it uses program memory, rather than system memory. 

it can't be used for dynamic content, but for what you're using, it will help a lot.